src/Sql/Sql/VulnerabilityAssessment/Model/VulnerabilityAssessmentSettingsModel.cs

```diff
@@ -93,18 +93,12 @@ public void ValidateContent()
                 NotificationEmail = NotificationEmail.Where(email => !string.IsNullOrEmpty(email)).ToArray();
 
                 // Validity checks:
-                // 1. Check that EmailAddresses are in correct format 
+                // Check that EmailAddresses are in correct format 
                 bool areEmailAddressesInCorrectFormat = Util.AreEmailAddressesInCorrectFormat(NotificationEmail);
                 if (!areEmailAddressesInCorrectFormat)
                 {
                     throw new Exception(Properties.Resources.EmailsAreNotValid);
                 }
-
-                // 2. check that EmailAdmins is not False and NotificationEmail is not empty
-                if (!EmailAdmins && (NotificationEmail == null || !NotificationEmail.Any()))
-                {
-                    throw new Exception(Properties.Resources.NeedToProvideEmail);
```

Comment:
remove this resource - "NeedToProvideEmail"

Reply:
or is TD uses it?

Reply:
TD is using it